lib/id3.dart

```diff
@@ -304,19 +319,24 @@ class MP3Instance {
 }
 
 int parseSize(List<int> block, int major_v) {
-  assert(block.length == 4);
-
   int len;
   if (major_v == 4) {
+    assert(block.length == 4);
```

Comment:
I think we shouldn't be using assert statements in production. They are only useful during development and testing.

Reply:
It doesn't run in production. That's why I didn't know it was broken. So it's OK to keep it in my opinion, it's self-documenting.